hera_cal/tests/test_abscal.py

```diff
@@ -131,10 +145,78 @@ def test_gains2calfits(self):
         if os.path.exists(cfname):
             os.remove(cfname)
 
+    def test_wiener(self):
+        # test smoothing
+        d = hc.abscal.wiener(self.data, window=(5, 15), noise=None, medfilt=True, medfilt_kernel=(1, 13))
+        nt.assert_equal(d[(24,37,'xx')].shape, (60, 64))
+        nt.assert_equal(d[(24,37,'xx')].dtype, np.complex)
+        # test w/ noise
+        d = hc.abscal.wiener(self.data, window=(5, 15), noise=0.1, medfilt=True, medfilt_kernel=(1, 13))
+        nt.assert_equal(d[(24,37,'xx')].shape, (60, 64))
+        # test w/o medfilt
+        d = hc.abscal.wiener(self.data, window=(5, 15), medfilt=False)
+        nt.assert_equal(d[(24,37,'xx')].shape, (60, 64))
+        # test as array
+        d = hc.abscal.wiener(self.data[(24,37, 'xx')], window=(5, 15), medfilt=False, array=True)
+        nt.assert_equal(d.shape, (60, 64))
+        nt.assert_equal(d.dtype, np.complex)
+
+    def test_Baseline(self):
```

Comment:
This test seems incomplete. Can you make it show that it works on baselines created from different pairs of antennas? Also, can you show that it works on baselines that are similar to just within the tolerance and just outside of it?

Reply:
good call. done